gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 accompanist = "0.34.0"
 androidDesugarJdkLibs = "2.0.4"
 # AGP and tools should be updated together
-androidGradlePlugin = "8.4.0"
+androidGradlePlugin = "8.5.1"
```

Comment:
Does your change need this?
If yes, we should update it in a distinct review.
If no, discard this change.

Reply:
- https://github.com/android/nowinandroid/pull/1536

Reply:
Discarded